From b3dcd24b8a186093cd821697b191080b8d53915b Mon Sep 17 00:00:00 2001
From: Oswald Buddenhagen <oswald.buddenhagen@gmx.de>
Date: Wed, 9 Aug 2023 19:15:31 +0200
Subject: [PATCH 1912/2871] t9001: remove excessive GIT_SEND_EMAIL_NOTTY=1

This was added by 3ece9bf0f9 (send-email: clear the $message_id after
validation, 2023-05-17) for no apparent reason, as this is required only
in cases when git's stdin is (must be) redirected, which isn't the case
here.

Signed-off-by: Oswald Buddenhagen <oswald.buddenhagen@gmx.de>
Signed-off-by: Junio C Hamano <gitster@pobox.com>
---
 t/t9001-send-email.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/t/t9001-send-email.sh b/t/t9001-send-email.sh
index 2051103226..2020e0ce69 100755
--- a/t/t9001-send-email.sh
+++ b/t/t9001-send-email.sh
@@ -592,7 +592,6 @@ test_expect_success $PREREQ 'clear message-id before parsing a new message' '
 	clean_fake_sendmail &&
 	echo true | write_script my-hooks/sendemail-validate &&
 	test_config core.hooksPath my-hooks &&
-	GIT_SEND_EMAIL_NOTTY=1 \
 	git send-email --validate --to=recipient@example.com \
 		--smtp-server="$(pwd)/fake.sendmail" \
 		$patches $threaded_patches &&
-- 
2.39.3 (Apple Git-145)

